src/pages/index.js

```diff
@@ -111,7 +111,7 @@ const HomePage = ({ data }) => {
               description={t('home.welcome.t2.description')}
             />
             <WelcomeTile
-              to="/docs/alerts/new-relic-alerts/getting-started/introduction-new-relic-alerts"
+              to="/docs.newrelic.com/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts#view-lookout"
```

Comment:
```suggestion
              to="/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts#view-lookout"
```

I'll let the docs team take a look at the updated title and link, but happened to noticed the URL is a bit funky here since this is a relative link to another page within the docs site. The suggestion above should fix it.